guides/serialization_and_saving.py

```diff
@@ -386,7 +386,7 @@ def from_config(cls, config):
 
 In order to save/load a model with custom-defined layers, or a subclassed model,
 you should overwrite the `get_config` and optionally `from_config` methods.
-Additionally, you should use register the custom object so that Keras is aware of it.
```

Comment:
This one is indeed a typo -- please restrict the PR to this typo, and fix it in the `md` and `ipynb` files also. Thank you!